demo/paging/mock-server-results-service.ts

```diff
@@ -3,7 +3,7 @@ import {Observable} from "rxjs";
 import {PagedData} from "./model/paged-data";
 import {CorporateEmployee} from "./model/corporate-employee";
 import {Page} from "./model/page";
-var companyData = require('../../assets/data/company.json');
+var companyData = require('../../assets/data/forRowGrouping.json');
```

Comment:
Actually there is no need for an exclusive service as the Demo for Row Grouping is grabbing the file directly.